### PR TITLE
Fix deprecated Api calls and other minor fixes.

### PR DESCRIPTION
**Pull request by smarthi**

### CODE REVIEW

**Comment by smarthi** [comment]
Needs 'Distributed testing' on Storm, MapReduce and Spark before merging this PR to master.

File: src/main/java/org/apache/pirk/responder/wideskies/storm/EncColMultBolt.java [hunk 1]
```diff
@@ -103,13 +101,13 @@ public void execute(Tuple tuple)
...
-      colVal1 = (BigInteger) tuple.getValueByField(StormConstants.ENCRYPTED_VALUE_FIELD);
+      BigInteger colVal1 = (BigInteger) tuple.getValueByField(StormConstants.ENCRYPTED_VALUE_FIELD);
 
       logger.debug("Received tuple in ECM, multiplying {} to col {}", colVal1, colIndex);
 
       if (resultsMap.containsKey(colIndex))
       {
-        colMult = colVal1.multiply(resultsMap.get(colIndex));
+        BigInteger colMult = colVal1.multiply(resultsMap.get(colIndex));
         resultsMap.put(colIndex, colMult.mod(nSquared));
```
Comment (clharris) [inline src/main/java/org/apache/pirk/responder/wideskies/storm/EncColMultBolt.java]: Will initializing colVal1 and colMult inside execute lead to extra garbage collection time? I mean not much per instance, but execute gets called billions of times.

Comment (smarthi) [inline src/main/java/org/apache/pirk/responder/wideskies/storm/EncColMultBolt.java]: I don't think it changes the initialization, even earlier the initialization was happening in the execute().

**Comment by ellisonanne** [comment]
Looks good on initial review - I will run it through the distributed test suite and report back

**Comment by ellisonanne** [comment]
Discovered a bug that was recently introduced in the code (PIRK-61) while testing this PR - am fixing and will push a PR in a few - with the changes that will be in the PIRK-61 PR, this PR passes all distributed tests. I will go ahead and merge.

**Pull request closed without merging**